doc/stdlib/parallel.rst

```diff
@@ -374,6 +377,11 @@ General Parallel Computing Support
             body
         end
 
+.. function:: @everywhere
+
+    Executes an expression as is on all processors. In the event of exceptions on
```

Comment:
Maybe remove "as is" here, and add the caveats about closures to the docs for `@async` and `@spawn`.